packages/react/src/date-picker/docs/overview.mdx

```diff
@@ -1,49 +1,47 @@
+
+For an example of using this component in a form built with `react-hook-form` and `yup`, please see the [Single-page form template](/patterns/single-page-form).
+
 <DoHeading />
 
-- allow year or date navigation
-- use date format dd/mm/yyyy
-- allow approximate date options
-- use autocomplete where it might help with input—for example, date of birth
-- ensure users can use Tab to select a date
-- Make sure error messages are specific about what input is required. For example, dd/mm/yyyy should always be displayed in hint text.
+- always display the date format above the text input
+- use Australian date format 'dd/mm/yyyy'
+- allow users to navigate dates via the calendar
+- ensure users can enter dates via the calendar or text input
+- Make sure error messages are specific about what input is required.
 
 <DontHeading />
 
+- never hide the date format
 - use international date formats such as USA mm/dd/yyyy
-- use Date Select for historical dates such as birthdates – use Date Input
 - hijack built-in keyboard navigation behaviour.
 
-## Date picker
+## Tracking the input value
 
-`DatePicker` is a [controlled component](https://reactjs.org/docs/forms.html#controlled-components) which means consumers of this component need to manage the state of this component by using the `value`, `onChange` and `onInputChange` props.
-
-For an example of using this component in a form built with `react-hook-form` and `yup`, please see the [Single-page form template](/patterns/single-page-form).
-
-```jsx live
-() => {
-	const [value, setValue] = React.useState();
-	return <DatePicker label="Select date" value={value} onChange={setValue} />;
-};
-```
-
-### Tracking the input value
-
-The `DatePicker` component has two methods of input:
+The Date picker component has two methods of input:
 
 1. Typing in a date string (dd/mm/yyyy) via the text input
 2. Selecting a date via the calendar widget
 
 As we do not have any sort of input masking due to accessibility concerns, there is a possibilty for the user to enter an invalid value via the text input. In some scenarios, it can be useful to know what the value of the text input is so the user can be informed of any invalid values.
 
-For example, if the `DatePicker` component was placed inside of a form an optional field. If the user typed in an invalid value via the text input, the `onChange` event would never fire. This may lead the user to believe what they have typed is valid when actually their input will not be saved.
+For example, if the Date picker component was placed inside of a form an optional field. If the user typed in an invalid value via the text input, the `onChange` event would never fire. This may lead the user to believe what they have typed is valid when actually their input will not be saved.
```

Comment:
```suggestion
For example, if the Date picker component was an optional field. If a user inputs an invalid value via the text input, the `onChange` event will never fire. This may lead the user to believe what they have typed is valid when actually their input will not be saved.
```